docs/rabbitmq.config.example

```diff
@@ -288,19 +297,37 @@
    %% NB: Change these only if you understand what you are doing!
    %%
 
+   %% Queue master location strategy:
+   %%   * <<"min-masters">>
+   %%   * <<"client-local">>
+   %%   * <<"random">>
+   %% See https://www.rabbitmq.com/ha.html#queue-master-location
+   %%
+   %% {queue_master_locator, <<"client-local">>},
+
+   %% Batch size of messages to synchronise between queue mirrors
+   %% See https://www.rabbitmq.com/ha.html#batch-sync
+   %%
+   %% {mirroring_sync_batch_size, 4096},
+
+   %% Controls flow control between queue mirrors
+   %% Be careful with this configuration, it's controversial
```

Comment:
It is not controversial and disabling it is dangerous (as is not having any flow control) with well know outcomes: masters outpacing mirrors at some point, making them consume more and more RAM.